rll/setup_automatic_upgrade.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Add entry in /etc/cron.d/ to daily check and excute an upgrade for rightlink.
```

Comment:
at what time of day does that execute? we must ensure that not all servers do it at the same time...